android/quest/src/main/java/org/smartregister/fhircore/quest/ui/register/RegisterFragment.kt

```diff
@@ -69,7 +69,7 @@ class RegisterFragment : Fragment(), OnSyncListener, Observer<QuestionnaireSubmi
 
   @Inject lateinit var syncListenerManager: SyncListenerManager
 
-  private val appMainViewModel by activityViewModels<AppMainViewModel>()
+  val appMainViewModel by activityViewModels<AppMainViewModel>()
```

Comment:
`protected` visibility could also have worked here if the reason was for using it in the **RegisterFragmentTest** class. Additional annotation with `@VisibileForTesting` would be good